tests/Controllers/MagicLinkTest.php

```diff
@@ -57,4 +64,51 @@ public function testShowValidateErrorsInMagicLink(): void
 
         $result->assertSessionHas('errors', $expected);
     }
+
```

Comment:
Maybe it will help in the future `https://github.com/codeigniter4/shield/issues/465`

Reply:
Done.